feature_add_place/src/main/java/com/jyp/feature_add_place/presentation/PlaceMapActivity.kt

```diff
@@ -0,0 +1,344 @@
+package com.jyp.feature_add_place.presentation
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import com.jyp.feature_add_place.data.model.SearchPlaceResultModel
+import com.jyp.jyp_design.R
+import com.jyp.feature_add_place.databinding.ActivityPlaceMapBinding
+import com.jyp.feature_add_place.presentation.SearchPlaceActivity.Companion.SEARCH_PLACE_RESULT
+import com.jyp.jyp_design.resource.JypColors
+import com.jyp.jyp_design.ui.button.ButtonColorSetType
+import com.jyp.jyp_design.ui.button.ButtonType
+import com.jyp.jyp_design.ui.button.JypTextButton
+import com.jyp.jyp_design.ui.text.JypText
+import com.jyp.jyp_design.ui.text_input.JypTextInput
+import com.jyp.jyp_design.ui.text_input.TextInputType
+import com.jyp.jyp_design.ui.typography.type.TextType
+import net.daum.mf.map.api.MapPoint
+import net.daum.mf.map.api.MapView
+
+
+class PlaceMapActivity : ComponentActivity() {
+
+    private var _viewBinding: ActivityPlaceMapBinding? = null
+    private val viewBinding get() = _viewBinding!!
```

Comment:
```suggestion
    private val viewBinding: ActivityPlaceMapBinding by lazy {
        ActivityPlaceMapBinding.inflate(layoutInflater)
    }
```
backing property 대신 이렇게 하면 더 간단해보일것 같아